backends/imgui_impl_win32.cpp

```diff
@@ -126,6 +138,25 @@ bool    ImGui_ImplWin32_Init(void* hwnd)
     if (!::QueryPerformanceCounter((LARGE_INTEGER*)&perf_counter))
         return false;
 
+    // Load WGL symbols explicitly
+    // This is needed to allow compilation if and when non-OpenGL APIs are used with Win32.
+    // The function pointer names are preceded with an underscore ("_") to prevent duplicate
+    // symbols in projects where OpenGL32.dll is linked implicitly.
+    if (hglrc)
+    {
+        libgl = LoadLibraryA("opengl32.dll");
```

Comment:
nit: I don't think it's actually necessary, but for the sake of consistency with the rest of the file this (and all the other Windows library function calls) should be prefixed with `::`.